README.md

```diff
@@ -159,3 +159,4 @@ viewing the `import-mailbox-to-gmail.log` file.
 * To import mail from Apple Mail.app, make sure you export it first - the raw Apple Mail files can't be imported. You can export a folder by right clicking it in Apple Mail and choosing "Export Mailbox".
 * This script can import nested folders. In order to do so, it is necessary to preserve the email folders' hierarchy when exporting them as mbox files. In Apple Mail.app, this can be done by expanding all subfolders, selecting both parents and subfolders at the same time, and exporting them by right clicking the selection and choosing "Export Mailbox". 
 * If any of the folders have a ".mbox" extension and a file named "mbox" in them, the contents of the "mbox" file will be imported to the label named as the folder. This is how Apple Mail exports are structured.
+* To run under [Docker](https://www.docker.com/): First build the image, `docker build -t google/import-mailbox-to-gmail .`; then run the import command, `docker run --rm -it -v "/local/path/to/auth.json:/auth.json" -v "/local/path/to/mbox:/mbox" google/import-mailbox-to-gmail --json "/auth.json" --dir "/mbox"`
```

Comment:
1. I suggest breaking it down to make it more readable. Note that the syntax in my suggestion might be a bit off. :) I suggest adding numbered steps and putting each example command in its own line.

```suggestion
* To run under [Docker](https://www.docker.com/):
   1. Build the image:
       `docker build -t google/import-mailbox-to-gmail .`
   2. Run the import command:
     `docker run --rm -it -v "/local/path/to/auth.json:/auth.json" -v "/local/path/to/mbox:/mbox" google/import-mailbox-to-gmail --json "/auth.json" --dir "/mbox"`
```

2. Please explain the `/path/to/file.json:file.json` syntax in the README, as it it's unusual compared to running it normally from the terminal.

Reply:
I completely agree. I just didn't know if this should be more of an after thought / side note, or take up more space in the README. I've expanded it out a bit.